tests/framework/utils/k8s_helper.go

```diff
@@ -56,6 +57,7 @@ type K8sHelper struct {
 	remoteExecutor   *exec.RemotePodCommandExecutor
 	Clientset        *kubernetes.Clientset
 	RookClientset    *rookclient.Clientset
+	BucketClientset  *bktclient.Clientset
```

Comment:
if it is not okay to add `bucketclient`, I will remove it here and define in the test file

Reply:
Yeah, I think it might be best to add this to the `TestClient` the same as the `BucketClient`. https://github.com/rook/rook/blob/master/tests/framework/clients/test_client.go

Reply:
IMO it make more sense to add here than in `TestClient{}`, `bucketclient` is similar to `rookclient` to execute apis directly on OBC and it requires `rest.Config`

Reply:
K8SHelper has all very low-level clients. What is appropriate for the K8SHelper, based on current context might be a low-level OBCClientset, but the overall of the `BucketClientset` I think should be in `TestClient` because it is much more than just a clientset. It has many more helper methods. I think adding the bucket clientset here would be good, and adding the core of `BucketClientset` with its helper methods to `TestClient` is the best thing.

Reply:
There is already `bucket client` as part of [TestClient](https://github.com/rook/rook/blob/master/tests/framework/clients/test_client.go#L35), here it more like low-level k8s operations like updating labels which can be easily with bucket client apis

Reply:
Unless this changed recently, I may have been misinterpreting something. This looks good here as `BucketClientset  *bktclient.Clientset` with other stuff in `TestClient`.